TUM Campus App/Room Finder/RoomFinderViewController.swift

```diff
@@ -41,6 +47,18 @@ final class RoomFinderViewController: UITableViewController, UISearchResultsUpda
             return cell
         }
     }
+    
+    /// Shows an error message stating that we were unable to find the room.
+    /// - parameter roomName: The name of the room we were searching for. `nil` to remove the error message
+    private func showUnableToFindRoomErrorMessage(roomName: String?) {
+        guard let roomName = roomName else {
+            tableView.backgroundView = nil
+            return
+        }
+        tableView.backgroundView = errorMessageLabel
+        errorMessageLabel.text = NSString(format: "Unable to find room".localized as NSString, roomName) as String
```

Comment:
Wouldn't it be nice to do something like that:  `errorMessageLabel.text = "\("Unable to find room".localized) roomName"` instead of using `NSString`?

Reply:
because the substitution happens at different places within the string:
- "Unable to find room 'xx.xx.xxx'"
- "Raum 'xx.xxx.xx' konnte nicht gefunden werden".

There's no german phrasing I can think of where it'd sound natural having the room number at the end of the sentence.

Also, if there's additional languages being added, these translations too would have to be phrased in a way where the room number is at the end.

Reply:
Also using `NSString` ?should? effectively be a noop, since it'll share the storage from the `String`, this is really just about getting the formatting api

Reply:
Cool. I didn't think of that. You are right